### PR TITLE
Rename webthing-esp8266 to webthing-arduino.

### LINKED ISSUE #28: Rename webthing-esp8266 to webthing-arduino
As per https://github.com/mozilla-iot/webthing-arduino/releases/tag/v0.3.0

### PR DESCRIPTION
**Pull request by mrstegeman**
Fixes #28

### CODE REVIEW

File: things/index.html [hunk 1]
```diff
@@ -93,37 +91,44 @@ <h2>Installation</h2>
     &lt;/dependency&gt;
 &lt;/dependencies&gt;
 </code></pre>
-              </p>
```
Comment (benfrancis) [inline things/index.html]: Why did you remove all the closing paragraph tags?

File: things/index.html [hunk 2]
```diff
@@ -751,38 +748,38 @@ <h2>Example</h2>
...
             </section>
           </div>
         </section>
 
         <section id="libraries">
           <h2>Find Out More</h2>
           <div class="library">
-            <h3>webthing-node</h3>
-            <img src="../contribute/images/github.png" />
+            <h3>Node.js</h3>
```
Comment (benfrancis) [inline things/index.html]: I'm fine with renaming them, but something funky is going on with the layout and I can't tell why...

**Comment by benfrancis** [comment]
Are you seeing this too?

Before:
screenshot from 2018-07-06 19-35-08


After:
screenshot from 2018-07-06 19-34-37

**Comment by benfrancis** [comment]
Oh I bet it's just because I'm loading it over file:// and the fonts don't work :P

**Comment by benfrancis** [comment]
Yeah not as bad with the right fonts, but something weird is still going on.

screenshot from 2018-07-06 19-42-49

File: things/index.html [hunk 3]
```diff
@@ -751,38 +748,38 @@ <h2>Example</h2>
...
             <a class="button" href="https://github.com/mozilla-iot/webthing-python">Learn More</a>
           </div>
           <div class="library">
-            <h3>webthing-java</h3>
-            <img src="../contribute/images/github.png" />
+            <h3>Java</h3>
+            <img alt="GitHub logo" src="../contribute/images/github.png" />
             <a class="button" href="https://github.com/mozilla-iot/webthing-java">Learn More</a>
           </div>
+          <br>
```
Comment (benfrancis) [inline things/index.html]: Ah, that's it. What is this for?

Comment (mrstegeman) [reply things/index.html]: Oops. That line was gonna be part of another PR.

Comment (benfrancis) [reply things/index.html]: Ah!

**Comment by mrstegeman** [comment]
@benfrancis Fixed.

File: things/index.html (hunk 1 above)
Comment (mrstegeman) [reply things/index.html]: My linter was complaining. I can add back.

**Comment by mrstegeman** [comment]
@benfrancis Comments all addressed.

**Comment by benfrancis** [comment]
Thanks!
